specification/devopsinfrastructure/Microsoft.DevOpsInfrastructure.Management/main.tsp

```diff
@@ -21,10 +21,10 @@ namespace Microsoft.DevOpsInfrastructure;
 
 /** Api versions */
 enum Versions {
-  /** 2024-10-19 version */
+  /** 2025-01-21 version */
   @useDependency(Azure.ResourceManager.Versions.v1_0_Preview_1)
   @useDependency(Azure.Core.Versions.v1_0_Preview_2)
-  `2024-10-19`,
+  `2025-01-21`,
```

Comment:
Can we directly update the stable version in-place? Do we have any rules to limit this?

Reply:
yes:

https://github.com/Azure/azure-rest-api-specs/blob/8384a760a78bafed9ca23cd6eb918ba21f780ebb/specification/suppressions.yaml#L32

Reply:
This caused an un-necessary breaking for us, @mikeharder do we have any plan to mitigate the impact for existing released versions?

Reply:
@MaryGao: Can you elaborate on the question?  Are you asking for all the suppressions of existing specs to be resolved, either by removing the swagger, or adding back the typespec sources?  Or something else?